src/backend/functions/teams.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Handler } from '@netlify/functions';
+import { Description_Bullet, Prisma, PrismaClient, WBS_Element } from '@prisma/client';
+import {
+  ApiRoute,
+  ApiRouteFunction,
+  apiRoutes,
+  API_URL,
+  buildServerFailureResponse,
+  buildSuccessResponse,
+  routeMatcher,
+  Team,
+  WbsNumber,
+  DescriptionBullet
+} from 'utils';
+
+const prisma = new PrismaClient();
+
+const relationArgs = Prisma.validator<Prisma.TeamArgs>()({
+  include: {
+    members: true,
+    leader: true,
+    projects: {
+      include: {
+        wbsElement: true
+      }
+    }
+  }
+});
+
+const wbsNumOf = (element: WBS_Element): WbsNumber => ({
+  carNumber: element.carNumber,
+  projectNumber: element.projectNumber,
+  workPackageNumber: element.workPackageNumber
+});
+
+export const descBulletConverter = (descBullet: Description_Bullet): DescriptionBullet => ({
+  ...descBullet,
+  id: descBullet.descriptionId,
+  dateDeleted: descBullet.dateDeleted ?? undefined
+});
+
+const teamsTransformer = (team: Prisma.TeamGetPayload<typeof relationArgs>): Team => {
+  if (team === null) throw new TypeError('Team not found');
+
+  return {
+    teamId: team.teamId ?? undefined,
```

Comment:
Why `?? undefined`? Isn't this a field that's always required? Plus didn't we already check team being null, so at this point this field should be guaranteed to be there, right?

Reply:
true